neptune/new/project.py

```diff
@@ -157,3 +157,204 @@ def _as_list(
+
+        `.stop()` will be automatically called when a script that initialized the connection finishes
+        or on the destruction of Neptune context.
+
+        When using Neptune with Jupyter notebooks it's a good practice to stop the connection manually as it
+        will be stopped automatically only when the Jupyter kernel stops.
+
+        Args:
+            seconds (int or float, optional): Seconds to wait for all tracking calls to finish
+                before stopping the tracked run.
+                If `None` will wait for all tracking calls to finish. Defaults to `True`.
+
+        Examples:
+            If you are initializing the connection from a script you don't need to call `.stop()`:
+
+            >>> import neptune.new as neptune
+            >>> project = neptune.init_project(name='MY_WORKSPACE/MY_PROJECT')
+
+            >>> # Your training or monitoring code
+            ... pass
+            ... # If you are executing Python script .stop()
+            ... # is automatically called at the end for every Neptune object
+
+            If you are initializing multiple connection from one script it is a good practice
+            to .stop() the unneeded connections. You can also use Context Managers - Neptune
+            will automatically call .stop() on the destruction of Project context:
+
+            >>> import neptune.new as neptune
+
+            >>> # If you are initializing multiple connections from the same script
+            ... # stop the connection manually once not needed
+            ... for project_name in projects:
+            ...   project = neptune.init_project(name=project_name)
+            ...   # Your training or monitoring code
+            ...   pass
+            ...   project.stop()
+
+            >>> # You can also use with statement and context manager
+            ... for project_name in projects:
+            ...   with neptune.init_project(name=project_name) as project:
+            ...     # Your code
+            ...     pass
+            ...     # .stop() is automatically called
+            ...     # when code execution exits the with statement
+
+        .. warning::
+            If you are using Jupyter notebooks for connecting to a project you need to manually invoke `.stop()`
+            once the connection is not needed.
+
+        You may also want to check `stop docs page`_.
+
+        .. _stop docs page:
+            https://docs.neptune.ai/api-reference/project#.stop
+        """
+        AttributeContainer.stop(self, seconds=seconds)
+
+    def get_structure(self) -> Dict[str, Any]:
+        """Returns a project's metadata structure in form of a dictionary.
+
+        This method can be used to traverse the project's metadata structure programmatically
+        when using Neptune in automated workflows.
+
+        .. danger::
+            The returned object is a deep copy of an internal project's structure.
```

Comment:
-> The returned object is a shallow copy of an internal structure. Any modifications to it may result in tracking malfunction.